rust/otap-dataflow/crates/admin/src/telemetry.rs

```diff
@@ -98,6 +98,9 @@ pub struct MetricsQuery {
     /// Output format: json (default), json_compact, line_protocol, prometheus
     #[serde(default)]
     format: Option<OutputFormat>,
+    /// When true, metric set which have all zero values are kept in the output. Default: false.
+    #[serde(default = "default_false")]
```

Comment:
I introduced this flag to monitor unused channels. This is useful for analyzing DAG creation and detecting inactive subgraphs in the DAG.